### PR TITLE
Added Parth Dwivedi card information

### PR DESCRIPTION
**Pull request by Parth4git**
I Have Added Parth Dwivedi  card information which includes all essential information's

### CODE REVIEW

**Comment by Parth4git** [comment]
@Syknapse pls review it

File: index.html [hunk 1]
```diff
@@ -1389,6 +1389,34 @@ <h2>A project for first-time contributions</h2>
         </div>
          <!-- ________ You-Know-Who card END ________  -->
 
+
+          <!-- ________ Parth Dwivedi card START ________  -->
+        <div class="card">
+          <p class="name">Parth Dwivedi</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com" target="_blank">@_Parth_Dwivedi_</a>
```
Comment (ascodeasice) [inline index.html:1398]: Hi, please note that you should change the link of your profile account, so it's not the website of main page.
You should change: 
- [ ] twitter link
-  [ ] github link

```suggestion
            <a href="https://www.twitter.com/{your username}" target="_blank">@_Parth_Dwivedi_</a>
```

**Comment by Parth4git** [comment]
I have changed the link of your profile accounts as you mentioned now please merge it @ascodeasice

**Comment by ascodeasice** [comment]
Thank you for your contribution

**Pull request merged**